scripts/check_if_pr_is_low_risk.py

```diff
@@ -110,7 +110,8 @@ def load_diff(
             return [], {}
         elif len(split) == 2:
             diff_files.append((split[1], split[1]))
-        elif len(split) == 3:
+        # Case when len(split) == 3.
+        else:
```

Comment:
ditto: Keep the `elif` and instead raise an exception in the `else` block. Alternatively, assert that `len(split) == 3` at the start of this `else` block

Reply:
Gone with the latter.